Airports/EGLC/Stars.txt

```diff
@@ -14,8 +14,8 @@ STAR:EGLC:27:GODLU1F:NEVIL OSPOL NETVU SOXUX OKVAP GODLU ELMIV LCE11 LCE12 LCE13
 STAR:EGLC:09:GODLU1F:NEVIL OSPOL NETVU SOXUX OKVAP GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV LCS01 LCS02 TODBI ODLEG
 STAR:EGLC:27:JACKO1H:HON ROGBI TIXEX ODVOD ROPMU NUDNA INLIM JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 TOPDU LAVNO
 STAR:EGLC:09:JACKO1H:HON ROGBI TIXEX ODVOD ROPMU NUDNA INLIM JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV LCS01 LCS02 TODBI ODLEG
-STAR:EGLC:27:JACKO1L:WAL LISTO PEDIG ROGBI TIXEX ODVOD ROPMU NUDNA INLIM JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 TOPDU LAVNO
-STAR:EGLC:09:JACKO1L:WAL LISTO PEDIG ROGBI TIXEX ODVOD ROPMU NUDNA INLIM JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV LCS01 LCS02 TODBI ODLEG
+STAR:EGLC:27:JACKO2L:LISTO PEDIG ROGBI TIXEX ODVOD ROPMU NUDNA INLIM JACKO
+STAR:EGLC:09:JACKO2L:LISTO PEDIG ROGBI TIXEX ODVOD ROPMU NUDNA INLIM JACKO
```

Comment:
Please add the transition to 09: NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV LCS01 LCS02 TODBI ODLEG